packages/features/bookings/components/event-meta/Locations.tsx

```diff
@@ -9,13 +9,31 @@ import { EventMetaBlock } from "./Details";
 export const EventLocations = ({ event }: { event: PublicEvent }) => {
   const { t } = useLocale();
   const locations = event.locations;
+
   if (!locations?.length) return null;
 
+  const getLocationToDisplay = (location: PublicEvent["locations"][number]) => {
+    const locationType = getEventLocationType(location.type);
+    const defaultLocationLabel = t(locationType?.label ?? "");
+
+    switch (locationType?.label) {
+      case "in_person":
+        return location?.address ?? defaultLocationLabel;
```

Comment:
if "show of booking page" is not checked we only display the defaultLocationLabel which in this case would be In Person (Organizer Address)

Reply:
nice job

Reply:
@Udit-takkar We should use the same logic as in https://github.com/calcom/cal.com/blob/main/apps/web/components/booking/AvailableEventLocations.tsx#L26 

There seems to be something else as well that's missing.

Reply:
it used to show a tooltip on hover too, for long named locations that overflow

Reply:
@hariombalhara wanna add the patch yourself or wait for @Udit-takkar ?

Reply:
@PeerRich  fixed now